### PR TITLE
Chore/standards again

### PR DESCRIPTION
**Pull request by kaisermann**
here we go again

### CODE REVIEW

File: packages/micro-core/index.ts [hunk 1]
```diff
@@ -1,2 +1,2 @@
 export * from './lib/index'
-export * from './plugins/index'
+// export * from './plugins/index'
```
Comment (kaisermann) [inline packages/micro-core/index.ts:2]: Didn't know what to do here

File: packages/micro-core/plugins/index.ts [hunk 2]
```diff
@@ -1 +1 @@
-export * from './build'
+// export * from './build'
```
Comment (kaisermann) [inline packages/micro-core/plugins/index.ts:1]: Also didn't know what to do here

File: packages/micro-react/components/once.ts [hunk 3]
```diff
@@ -7,7 +7,7 @@ export const once = <T extends (...args: any) => any>(fn: T) => {
       return ret as ReturnType<T>
     }
     called = true
-    ret = fn(...args as any)
+    ret = fn(...(args as any))
```
Comment (kaisermann) [inline packages/micro-react/components/once.ts:10]: This may be semantically wrong

File: packages/micro-react-intl/index.ts [hunk 4]
```diff
@@ -1,2 +1,2 @@
 export * from './components'
-export * from './plugins/bundle'
+// export * from './plugins/bundle'
```
Comment (kaisermann) [inline packages/micro-react-intl/index.ts:2]: Also didn't know what to do here

**Review by tlgimenes — APPROVED** [review]
LGTM

File: packages/micro-core/index.ts (hunk 1 above)
Comment (tlgimenes) [reply packages/micro-core/index.ts:2]: you can safely remove this 👍

File: packages/micro-core/plugins/index.ts (hunk 2 above)
Comment (tlgimenes) [reply packages/micro-core/plugins/index.ts:1]: you can safely remove this 👍

File: packages/micro-react/components/once.ts (hunk 3 above)
Comment (tlgimenes) [reply packages/micro-react/components/once.ts:10]: ok 👍

File: packages/micro-react-intl/index.ts (hunk 4 above)
Comment (tlgimenes) [reply packages/micro-react-intl/index.ts:2]: you can safely remove this 👍

**Pull request merged**